web/app/package.json

```diff
@@ -68,7 +68,7 @@
     "webpack": "4.20.2",
     "webpack-bundle-analyzer": "^3.0.3",
     "webpack-cli": "3.0.7",
-    "webpack-dev-server": "3.1.14"
+    "webpack-dev-server": "^3.2.0"
```

Comment:
can we continue with the version pinning already here? we can pin to `3.3.1` rather than `^`

Reply:
No problem. I'll change to pin the version to 3.3.1.